server/guardrails/input_rail.py

```
@@ -172,29 +180,49 @@ def _build_rails_sync():
     return LLMRails(config=rails_config, llm=_build_llm())
 
 
+def _ensure_rails_in_thread():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_ensure_rails_in_thread`

(ANN202)